src/containers/Nodes/getNodesColumns.tsx

```diff
@@ -89,6 +89,7 @@ const versionColumn: NodesColumn = {
         return <CellWithPopover content={row.Version}>{row.Version}</CellWithPopover>;
     },
     sortable: false,
+    resizeable: true,
```

Comment:
Currently only `Version` column is resizeable, because proper resize also requires update for components used inside cells.

For example, `NodesHostWrapper` has fixed `330px` width , resize makes no sense for such component.

Other columns will be added further step by step, if needed